go.mod

```diff
@@ -386,10 +393,15 @@ replace (
 
 replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.golang v1.3.6-0.20220106130409-e28a1db639f8
 
+replace go.temporal.io/api => go.temporal.io/api v1.8.0
```

Comment:
Can we say why we need to pin these so it's not a mystery? 😄

Reply:
This also seemed to be an error on my end. I've removed the replace statement and it appears to work as expected